tests/animations/invalid/ellipse.json

```diff
@@ -0,0 +1,142 @@
+{
+"v": "5.5.7",
+"ip": 0,
+"op": 180,
+"nm": "Animation",
+"mn": "{8f1618e3-6f83-4531-8f65-07dd4b68ee2e}",
```

Comment:
Out of curiosity, do you know how you ended up with these `mn` UUID strings?  I've only seen human-readable Match Names so far.  Our layer effects implementation relies on some hard coded values, if would be good to know if there are changes in AE/BM.

Reply:
I'm not sure on that, I copied that out of docs/static/examples. Possibly from LottieFiles tooling. @mbasaglia do you know?